js/packages/openinference-core/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from "./trace";
+export * from "./utils";
```

Comment:
exporting utils here to get access to withSafety and safelyJSONStringify and safelyJSONParse - that, it also exports some commonly used type utils - if we want to handle this in a different way can switch

this will make https://github.com/Arize-ai/openinference/blob/main/js/packages/openinference-core/src/utils/index.ts#L10 available on this package